service/build.gradle

```diff
@@ -30,10 +30,19 @@ dependencies {
 	implementation 'ch.qos.logback.contrib:logback-json-classic:0.1.5'
 	implementation 'ch.qos.logback.contrib:logback-jackson:0.1.5'
 	implementation 'org.springframework.boot:spring-boot-starter-web'
+	implementation 'org.springframework:spring-jms:5.2.6.RELEASE'
+	implementation 'org.apache.qpid:qpid-jms-client:0.51.0'
+	implementation 'org.apache.httpcomponents:httpclient:4.5.12'
+	implementation 'junit:junit:4.12'
 	testImplementation 'org.springframework.boot:spring-boot-starter-test'
 	testImplementation "io.rest-assured:rest-assured:4.2.0"
 	testImplementation "io.rest-assured:json-path:4.2.0"
 	testImplementation "io.rest-assured:xml-path:4.2.0"
+	testImplementation "org.apache.activemq.tooling:activemq-junit:5.13.1"
+	testImplementation "org.apache.activemq:activemq-broker:5.16.0"
```

Comment:
Are these activemq dependencies actually used? The tests seem to use TestContainer instead.

Reply:
Removed, from previous attempt with queue tests